radiotherapyai/dicom/append.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2022 Radiotherapy AI Holdings Pty Ltd
```

Comment:
If this was copied from pymedphys, it should reference it, and presumably honour it's license...

Reply:
In this case the copyright in PyMedPhys was my own, when it's my own copyright I'm free to not reference previous times where I have written something similar...

Reply:
Pulled out into its own issue:

https://github.com/RadiotherapyAI/RadiotherapyAI/issues/75